packages/preview-button/admin/src/components/PreviewLink/index.css

```diff
@@ -0,0 +1,18 @@
+.utrecht-preview-dialog {
+  max-block-size: 70vh;
+  overflow-y: auto;
+  & .utrecht-preview-dialog__body {
+    --utrecht-space-around: 1;
+  }
+}
```

Comment:
Denk je dat het handig is om een dialog component te maken voor het Utrecht design system? Bijvoorbeeld op basis van wat we voor Signalen hebben gedaan, en misschien in samenwerking met de mensen van Acato/Tilburg?

https://codefornl.slack.com/archives/C01DAT4TRPF/p1737538080448069

- [ ]

Reply:
Ik heb er al eentje gebouwd in de [Strapi-repo](https://github.com/frameless/strapi/tree/main/packages/ui/src/components/ModalDialog). Als je het goed vindt, kan ik het in een aparte PR in deze plugin integreren.

Reply:
Ik heb onze Dialog component geïntegreerd! Het werkt veel beter.
<img width="1494" alt="Screenshot 2025-01-31 at 13 58 47" src="https://github.com/user-attachments/assets/653e3892-97b6-4d32-a0dd-23edaced319d" />

Reply:
Already, squashed the commit.